crates/subspace-farmer/src/single_disk_plot/plotting.rs

```diff
@@ -0,0 +1,133 @@
+use crate::rpc_client::RpcClient;
+use crate::single_disk_plot::{PlottingError, SectorMetadata};
+use bitvec::order::Lsb0;
+use bitvec::prelude::*;
+use parity_scale_codec::Encode;
+use rayon::prelude::*;
+use std::num::NonZeroU16;
+use std::sync::atomic::{AtomicBool, Ordering};
+use subspace_core_primitives::crypto::kzg::Witness;
+use subspace_core_primitives::{Piece, PieceIndex, PublicKey, SectorId, PIECE_SIZE};
+use subspace_solving::derive_chunk_otp;
+use tokio::runtime::Handle;
+use tracing::{debug, error};
+
+/// Plotting status
+pub enum PlottingStatus {
```

Comment:
I was thinking about that. The only reason is because `Result<PlottingResult, PlottingError>` looks weird :slightly_smiling_face: I'm open for suggestions with better naming here.

Reply:
Then `PlottingOutcome`?